schemas/alpha/azure.yaml.json

```diff
@@ -541,16 +543,21 @@
                     "title": "The platform target",
                     "default": "amd64"
                 },
-                "tag": {
-                    "type": "string",
-                    "title": "The tag that will be applied to the built container image.",
-                    "description": "If omitted, a unique tag will be generated based on the format: {appName}/{serviceName}-{environmentName}:azd-deploy-{unix time (seconds)}. Supports environment variable substitution. For example, to generate unique tags for a given release: myapp/myimage:${DOCKER_IMAGE_TAG}"
-                },
                 "registry": {
                     "type": "string",
-                    "title": "The container registry to push the image to.",
+                    "title": "Optional. The container registry to push the image to.",
                     "description": "If omitted, will default to value of AZURE_CONTAINER_REGISTRY_ENDPOINT environment variable. Supports environment variable substitution."
                 },
+                "image": {
```

Comment:
It feels a little inconvenient to parse which are inputs (source artifacts) and outputs (deployment artifacts) at a glance:

```yaml
services:
  nginx:
    host: containerapp
    docker:
      image: nginx // This is input
      registry: my-registry // This is output
      tag: some-tag // This is output
```

I wonder if we should instead say: "services are either built from source, or can be a prebuilt artifact, i.e. image"

```yaml
services:
  nginx:
    host: containerapp
    image: nginx // Instead of `project`, we have `image`
    docker:
      registry: my-registry // This is output
      tag: some-tag // This is output
```